pkg/apis/pipeline/v1alpha1/resource_types.go

```diff
@@ -43,10 +43,13 @@ const (
 
 	// PipelineResourceTypePullRequest indicates that this source is a SCM Pull Request.
 	PipelineResourceTypePullRequest PipelineResourceType = "pullRequest"
+
+	// PipelineResourceTypeCloudEvent indicates that this source is a cloud event URI
+	PipelineResourceTypeCloudEvent PipelineResourceType = "cloudevent"
```

Comment:
nit: CamelCase for consistency "cloudEvent"

Reply:
Done